admin/includes/languages/english/specials.php

```diff
@@ -58,4 +57,4 @@
 define('TEXT_INFO_PRE_ADD_INTRO', 'On large databases, you may Manually Add a Special by the Product ID<br /><br />This is best used when the page takes too long to render and trying to select a Product from the dropdown becomes difficult due to too many Products from which to choose.');
 define('TEXT_PRE_ADD_PRODUCTS_ID', 'Please enter the Product ID to be Pre-Added: ');
 define('TEXT_INFO_MANUAL', 'Product ID to be Manually Added as a Special');
-?>
\ No newline at end of file
+?>

```

Comment:
Shouldn't add the `?>` at the end of the file.

Reply:
I can fix that, but this file wasn't included in the PR.

Reply:
... if the file wasn't in the PR, how does it show in the diff?

Reply:
I've removed it.